guides/common/modules/con_updating-project-to-next-minor-version.adoc

```diff
@@ -0,0 +1,6 @@
+[id="updating-project-to-next-minor-version_{context}"]
+= Updating {Project} to the Next Minor Version
+
+Use this guide to update your {ProjectServer} and {SmartProxyServer} to a new minor release version, such as from {ProjectVersion}.0 to {ProjectVersion}.1.
+
+The minor releases patch security vulnerabilities and minor issues discovered after code is released, and are often fast and non-disruptive to your operating environment.
```

Comment:
```suggestion
The minor releases are non-disruptive to your operating environment and often fast.

{Team} recommends performing updates regularly, because the minor releases patch security vulnerabilities and minor issues discovered after code is released.
```
@ehelms @ekohl Can we go ahead with such a recommendation?

Reply:
I am not sure I agree with `The minor releases are non-disruptive to your operating environment and often fast.` Mostly the first part, given the latter is covered by "often".

Reply:
@ehelms I'm looking for a motivation for the user to update at all and how often they should. Who could advise us on this?

Reply:
Addressed :+1: